### PR TITLE
docs: translation for passing data deeply with context

### LINKED ISSUE #384: Passing Data Deeply with Context
Page: https://id.react.dev/learn/passing-data-deeply-with-context

Assignee: @ridhlab

### PR DESCRIPTION
**Pull request by ridhlab**
Closes #384 

## Description

Translate the : **Passing Data Deeply with Context** page.
Page URL:  [https://id.react.dev/learn/passing-data-deeply-with-context](https://id.react.dev/learn/passing-data-deeply-with-context)

### CODE REVIEW

File: src/content/learn/passing-data-deeply-with-context.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
-But passing props can become verbose and inconvenient when you need to pass some prop deeply through the tree, or if many components need the same prop. The nearest common ancestor could be far removed from the components that need data, and [lifting state up](/learn/sharing-state-between-components) that high can lead to a situation called "prop drilling".
+Tapi mengoper *props* bisa menjadi bertele-tele dan tidak nyaman ketika Anda perlu mengoper beberapa *prop* secara mendalam melalui pohon, atau jika banyak komponen membutuhkan *prop* yang sama. Leluhur umum terdekat bisa jadi jauh dari komponen yang membutuhkan data, dan [memindahkan state ke atas](/learn/sharing-state-between-components) dapat menyebabkan yang disebut "*prop drilling*".
 
 <DiagramGroup>
 
 <Diagram name="passing_data_lifting_state" height={160} width={608} captionPosition="top" alt="Diagram with a tree of three components. The parent contains a bubble representing a value highlighted in purple. The value flows down to each of the two children, both highlighted in purple." >
 
-Lifting state up
+Memindahkan state ke atas
```
Comment (r17x) [inline src/content/learn/passing-data-deeply-with-context.md:30]:
```suggestion
Memindahkan *state* ke atas
```

File: src/content/learn/passing-data-deeply-with-context.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ Prop drilling
 
 </DiagramGroup>
 
-Wouldn't it be great if there were a way to "teleport" data to the components in the tree that need it without passing props? With React's context feature, there is!
+Bukankah lebih bagus jika ada cara untuk "memindahkan" data ke komponen di dalam pohon yang membutuhkannya tanpa harus mengoper *props*? Dengan fitur *context* React, ternyata ada!
```
Comment (r17x) [inline src/content/learn/passing-data-deeply-with-context.md:41]:
```suggestion
Bukankah lebih bagus jika ada cara untuk "memindahkan" data ke komponen di dalam pohon (*tree*) yang membutuhkannya tanpa harus mengoper *props*? Dengan fitur *context* React, ternyata ada!
```

File: src/content/learn/passing-data-deeply-with-context.md [hunk 3]
```diff
@@ -38,11 +38,11 @@ Prop drilling
...
 
-## Context: an alternative to passing props {/*context-an-alternative-to-passing-props*/}
+## *Context*: sebuah alternatif untuk mengoper *props* {/*context-an-alternative-to-passing-props*/}
 
-Context lets a parent component provide data to the entire tree below it. There are many uses for context. Here is one example. Consider this `Heading` component that accepts a `level` for its size:
+*Context* memungkinkan sebuah komponen induk menyediakan data untuk seluruh pohon di bawahnya. Ada banyak kegunaan dari *context*. Berikut ini salah satu contohnya. Perhatikan komponen `Heading` ini yang menerima `level` untuk ukurannya:
```
Comment (r17x) [inline src/content/learn/passing-data-deeply-with-context.md:45]:
```suggestion
*Context* memungkinkan sebuah komponen induk menyediakan data untuk seluruh pohon (*tree*) di bawahnya. Ada banyak kegunaan dari *context*. Berikut ini salah satu contohnya. Perhatikan komponen `Heading` ini yang menerima `level` untuk ukurannya:
```

File: src/content/learn/passing-data-deeply-with-context.md [hunk 4]
```diff
@@ -476,7 +476,7 @@ export default function Section({ level, children }) {
 }
 ```
 
-This tells React: "if any component inside this `<Section>` asks for `LevelContext`, give them this `level`." The component will use the value of the nearest `<LevelContext.Provider>` in the UI tree above it.
+Ini memberitahu React: "jika ada komponen di dalam `<Section>` ini yang meminta `LevelContext`, berikan `level` ini." Komponen akan menggunakan nilai dari `<LevelContext.Provider>` terdekat di pohon UI tree di atasnya.
```
Comment (r17x) [inline src/content/learn/passing-data-deeply-with-context.md:479]:
```suggestion
Ini memberitahu React: "jika ada komponen di dalam `<Section>` ini yang meminta `LevelContext`, berikan `level` ini." Komponen akan menggunakan nilai dari `<LevelContext.Provider>` terdekat di pohon UI (*tree*) di atasnya.
```

**Comment by mhaidarhanif** [comment]
@ridhlab and @r17x can check and resolve the above requests.

If everything's fine @r17x can approve this.

**Review by r17x — COMMENTED** [review]
Sorry for late response & thanks for your contribution @ridhlab

**Review by r17x — APPROVED** [review]

**Pull request merged**